wiring/src/spark_wiring_cellular.cpp

```diff
@@ -26,30 +26,18 @@
 namespace spark {
 
     CellularSignal CellularClass::RSSI() {
```

Comment:
RSSI no longer represents the info returned by this method. How about adding an alias for it, e.g. `Cellular.getSignal()`?